.github/workflows/ci.yml

```
@@ -0,0 +1,170 @@
+          export WHEELNAME=`ls dist/*.whl`
+          echo ${WHEELNAME}
+          pip install ${WHEELNAME}
+          cd tests
+
+      - name: Install extras
+        shell: bash
+        run: |
+          pip install .[plotting]
+
+      - name: Check licences of packages
+        run: |
+          pip install pip-licenses
+          pip-licenses
+          pip uninstall -y pip-licenses
+
+      - name: Test import
+        working-directory: tests
+        run: python -c "from ansys.dpf import post; print(post.Report(gpu=False))"
+
+      - name: Retrieve PyDPF-Post version
+        run: |
+          echo "::set-output name=VERSION::$(python -c "from ansys.dpf.post import __version__; print(__version__)")"
+          echo "PyDPF-Post version is: $(python -c "from ansys.dpf.post import __version__; print(__version__)")"
+        id: version
+
+      - name: Upload wheel to artifacts
+        uses: actions/upload-artifact@v2
+        with:
+          name: ${{ env.PACKAGE_NAME }}-v${{ steps.version.outputs.VERSION }}_wheel
+          path: ./dist/*
+
+      - name: Install OpenGL
+        shell: pwsh
+        run: |
+          Set-StrictMode -Version Latest
+          $ErrorActionPreference = "Stop"
+          $PSDefaultParameterValues['*:ErrorAction']='Stop'
+          git clone --depth 1 https://github.com/pyvista/gl-ci-helpers.git
+          powershell gl-ci-helpers/appveyor/install_opengl.ps1
+        if: runner.os == 'Windows'
+
+      - name: Install OS packages
+        run: |
+          sudo apt update
+          sudo apt install zip pandoc libgl1-mesa-glx xvfb
+        if: runner.os == 'Linux'
+
+      - name: Install offscreen rendering
+        shell: bash
+        run: |
+          ls -lh .ci
+          chmod +x .ci/setup_headless_display.sh
+          .ci/setup_headless_display.sh
+          pip install -r .ci/requirements_test_xvfb.txt
+          python .ci/display_test.py
+
+#      - name: Test offscreen rendering (Windows)
+#        shell: bash
+#        run:
+
+      - name: Install/Test Environment
+        run: |
+          pip install -r requirements_test.txt
+        if: always()
+
+      - name: Test API Docstrings
+        run: |
+           pytest --doctest-modules --junitxml=junit/test-doctests-results.xml ansys/dpf/post
+
+      - name: Kill all servers
+        shell: bash
+        run: |
+          echo $0
+          if pgrep -x "Ans.Dpf.Grpc" > /dev/null
+          then
+              pkill -f Ans.Dpf.Grpc.exe
+          fi
+        if: always()
+        continue-on-error: true
+
+      - name: Publish Doc Test Results
+        uses: actions/upload-artifact@v2
+        with:
+          name: ${{ env.PACKAGE_NAME }}_doctest
+          path: junit/test-doctests-results.xml
+        if: always()
+
+      - name: Test Post API
+        run: |
+          cd tests
+          pytest --cov=ansys.dpf.post --cov-report=xml --cov-report=html --log-level=ERROR --junitxml=junit/test-results.xml --reruns 2 .
```

@PProfizi You can add the verbose option if you want to get detailed info when test failure occurs.
```suggestion
          pytest -v --cov=ansys.dpf.post --cov-report=xml --cov-report=html --log-level=ERROR --junitxml=junit/test-results.xml --reruns 2 .
```

---

Will do, thx!